velox/functions/prestosql/aggregates/PrestoHasher.cpp

```diff
@@ -233,8 +246,11 @@ void PrestoHasher::hash<TypeKind::ARRAY>(
     BufferPtr& hashes) {
   auto baseArray = vector_->base()->as<ArrayVector>();
   auto indices = vector_->indices();
+  auto decodedRows = SelectivityVector(rows.size(), true);
+  decodedSelectivityVector(rows, vector_, decodedRows);
```

Comment:
We dont need to remove null rows here as `functions::toElementRows` already accounts for nulls in the dictionary and will skip those elements